src/doc/rustdoc/src/unstable-features.md

```diff
@@ -56,88 +56,6 @@ It is also not emitted for foreign items, aliases, extern crates and imports.
 These features operate by extending the `#[doc]` attribute, and thus can be caught by the compiler
 and enabled with a `#![feature(...)]` attribute in your crate.
 
-### `#[doc(cfg)]`: Recording what platforms or features are required for code to be present
```

Comment:
Shouldn't these docs be updated and moved to the stable part of the book instead of just deleted?

Reply:
On second thought, I think I'd prefer we land the implementation of the new features without stabilizing doc_cfg/doc_auto_cfg. Then we can check that it behaves as expected and stabilize soon after. It just seems risky to change so much at the same time as stabilizing.

Reply:
Erf. I suppose I badly rebased but it was supposed to be moved to the stable part.

And agreed on not stabilizing right away. One thing though: there are 3 features for this RFC, and `cfg_hide` at least it not used anymore. So I'm gonna put them all behind `doc_cfg` to make things easier.